blockstack-sdk/src/main/java/org/blockstack/android/sdk/ui/ConnectActivity.kt

```diff
@@ -0,0 +1,95 @@
+package org.blockstack.android.sdk.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.StyleRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.coroutineScope
+import kotlinx.android.synthetic.main.activity_connect.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.launch
+import org.blockstack.android.sdk.BlockstackConnect
+import org.blockstack.android.sdk.R
+
+class ConnectActivity : AppCompatActivity() {
```

Comment:
This Activity should probably be called `BlockstackConnectActivity`, so developers don't confuse it for some of their app Activities.